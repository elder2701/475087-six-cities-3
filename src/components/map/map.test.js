import React from "react";
import renderer from "react-test-renderer";
import createMapBlock from "./create-map-block.js";
import Map from "./map.jsx";

it(`<Map /> sould be render`, () => {
  createMapBlock();
  const tree = renderer
    .create(
        <Map
          offersCoords={[
            [51.38333, 6.9],
            [54.38333, 5.9]
          ]}
          name =''
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
