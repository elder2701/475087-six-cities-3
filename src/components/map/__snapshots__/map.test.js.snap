// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> sould be render 1`] = `
<section
  className=" map"
  id="map"
/>
`;
